orm/src/test/java/org/hibernate/search/test/configuration/Item.java

```diff
@@ -8,17 +8,16 @@
 
 import javax.persistence.CascadeType;
 import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import javax.persistence.ManyToOne;
 
 @Entity
 public class Item {
 
-	@Id @GeneratedValue
+	@Id
 	private Integer id;
 	private String description;
-	private Double price;
+	private Short price;
```

Comment:
It's to facilitate testing: for short (and byte) I can control String vs. numeric mapping via `@NumericField` (Double will be a numeric field by default, no way to disable it through `@NumericField`). Then I can assert on the right field type in the Lucene document.